funlet-call-me/callme.php.md

```diff
@@ -0,0 +1,125 @@
+## Annotated Source Code
```

Comment:
I think we can drop these from the code base. They were super useful for the implementation but I think long term they might just be confusing for the people looking at templates.

Reply:
That's a good point. When I started to create Twilio Functions from the selection of templates, I was under the impression that only a single script file was copied into my account? My mental model was: when I select a template, the script text is copy/pasted into my editor as a starting point. Is there more going on than meets the eye?

Reply:
I will remove them one by one, after I have completed the corresponding implementation.